src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -189,6 +191,12 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
       this.updateBreakpointAndResponsiveClass(this.flyoutWidth);
     }
 
+    if ((window.innerWidth - this.windowBufferSize) < this.config.minWidth) {
+      this.isFullscreen = true;
+    } else {
+      this.isFullscreen = false;
+    }
+
```

Comment:
Should we isolate this logic? I see it repeated down below... Something like `setFullscreen()` / `checkFullscreen()` etc...?

Reply:
Done